assets/blocks/course-outline/lesson-block/edit.js

```diff
@@ -120,14 +121,15 @@ const EditLessonBlock = ( props ) => {
 
 	return (
 		<>
-			<LessonBlockSettings { ...props } />
+			<LessonBlockSettings { ...props } fontSize={ fontSize } />
```

Comment:
Since props is already passed over, could this be just an extra `{ attributes: fontSize }` deconstruction in the `<LessonBlockSettings />`?

Reply:
Good idea! 61257b1c391f6c8ed58cfdc5c01b2a3cd6430871